velox/substrait/SubstraitToVeloxExpr.cpp

```diff
@@ -127,10 +127,34 @@ SubstraitVeloxExprConverter::toVeloxExpr(
       return toVeloxExpr(substraitExpr.selection(), inputType);
     case ::substrait::Expression::RexTypeCase::kCast:
       return toVeloxExpr(substraitExpr.cast(), inputType);
+    case ::substrait::Expression::RexTypeCase::kIfThen:
+      return toVeloxExpr(substraitExpr.if_then(), inputType);
     default:
       VELOX_NYI(
           "Substrait conversion not supported for Expression '{}'", typeCase);
   }
 }
 
+std::shared_ptr<const core::ITypedExpr>
+SubstraitVeloxExprConverter::toVeloxExpr(
+    const ::substrait::Expression_IfThen& substraitIfThen,
```

Comment:
Is this an IF expression or a SWITCH expression?

Can you extract this change into a separate PR and add a test?

Reply:
Done